src/main.cpp

```diff
@@ -381,6 +382,29 @@ int main ( int argc, char** argv )
             continue;
         }
 
+        // Disable delayed panning on startup ----------------------------------------
+        if ( GetFlagArgument ( argv,
+                               i,
+                               "--nodelayedpan", // no short form
+                               "--nodelayedpan" ) )
+        {
+            bDelayPan = false;
+            qInfo() << "- starting without delayed panning";
+            CommandLineOptions << "--nodelayedpan";
+            continue;
+        }
+
+        // Enable delayed panning on startup ----------------------------------------
+        if ( GetFlagArgument ( argv,
```

Comment:
Why do we need two cli flags? Wouldn't one be enough?

Reply:
Yes one would be enough. I realized this too after I made the PR. Two options would be useful should we change the default in a later release. We’d still need to decide on the default. I’d like _on_ as default in which case you need the `—nodelaypan` option. With the other default you want the `-P/—delayedpan` option.

Reply:
If switching it on means more processing power, I would not enable it by default (there is and was a lot going on with optimizing server performance and adding something which needs more power is counter productive). You can always switch it on of course.

Reply:
I remember that Volker mentioned that this delay pan might not be a better experience for everybody, for example bands. So I also wouldn't enable it per default.

Reply:
Maybe someone with a band could try it?